enhancements/installer/telemeter-metrics.md

```diff
@@ -0,0 +1,430 @@
+  * `version` 
+  * `platform`
+  * `le` - step size of 3 minutes with a max of 30
+
+An example of a sample tracking how long it took for bootstrap complete after the temporary control plane has come up: 
+```
+# HELP cluster_installation_bootstrap tracks the duration between the appearance of the temporary control plane and bootstrap completion.
+# TYPE cluster_installation_bootstrap histogram
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="3"} 0
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="6"} 0
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="9"} 0
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="12"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="15"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="18"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="21"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="24"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="27"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="30"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="+Inf"} 1
+cluster_installation_bootstrap_sum{command="create",platform="aws",result="success"} 10
+cluster_installation_bootstrap_count{command="create",platform="aws",result="success"} 1
+```
+
+#### Modification
+The `cluster_installation_modification`- metrics represent whether user-modified manifest or ignition-config assets were used to install the cluster.
+
+The proposed metrics represent categories of one or more asset file modified during an installation:
+
+Metric | Modified Files
+-------|---------------
+cluster_installation_modification_config_manifest | cloud-provider-config.yaml <br> cluster-config.yaml <br> cluster-infrastructure-02-config.yml <br> kube-cloud-config.yaml
+cluster_installation_modification_dns_manifest | cluster-dns-02-config.yml <br> cluster-ingress-02-config.yml
+cluster_installation_modification_network_manifest | cluster-network-01-crd.yml <br> cluster-network-02-config.yml <br> cluster-proxy-01-config.yaml
+cluster_installation_modification_scheduler_manifest | cluster-scheduler-02-config.yml
+cluster_installation_modification_cvo_manifest | cvo-overrides.yaml
+cluster_installation_modification_etcd_manifest | etcd-ca-bundle-configmap.yaml <br> etcd-client-secret.yaml <br> etcd-host-service-endpoints.yaml <br> etcd-host-service.yaml <br> etcd-metric-client-secret.yaml <br> etcd-metric-serving-ca-configmap.yaml <br> etcd-metric-signer-secret.yaml <br> etcd-namespace.yaml <br> etcd-service.yaml <br> etcd-serving-ca-configmap.yaml <br> etcd-signer-secret.yaml
```

Comment:
Do we not care about installer version for these metrics?

Also, can we not have a single metric `installer_asset_modified` with an `asset` label and value from `Asset.Name()`?  That way we could push this metric when we load each asset (and determine [whether is has been modified or not][1], instead of growing the more granular ability to track dirty files (vs. assets) on load and then iterating over dirty files to figure out which non-asset file groupings were modified or not.

[1]: https://github.com/openshift/enhancements/pull/205#discussion_r391313129